src/componentLibrary/Icon/Icon.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactComponent as CloseIcon } from '@material-icons/svg/svg/close/baseline.svg';
+import { ReactComponent as CheckmarkIcon } from '@material-icons/svg/svg/check/baseline.svg';
+import { SVGProps } from 'react';
+
+type IconVariant = 'close' | 'checkmark';
+
+interface IconProps extends SVGProps<SVGSVGElement> {
+  variant: IconVariant;
+}
+
+const Icon = ({ variant, ...props }: IconProps): JSX.Element | null => {
```

Comment:
Hi @tomjeatt. Can you be a little bit more specific please? Do you mean to use the pattern of conditionally rendering the icon in the wrapper instead of rendering component assigned as switch-case function result?

Reply:
Yes exactly that - I think the switch statement makes it harder to work out what's going on; returning `<CloseIcon />` or `<CheckMarkIcon />` is a bit more readable.

Reply:
Thanks for clarifying :)